sphinx-docs/installguide/install_mac.rst

```diff
@@ -0,0 +1,28 @@
+Mac Installation Guide
+===========================
+
+Macintosh installation uses the command line, but if you follow the instructions carefully, you should have no problem installing KA Lite.
+
+#. Download and install either Python2.6_ or Python2.7_ for Mac
+	.. _Python2.6: https://www.python.org/download/releases/2.6/
+	.. _Python2.7: https://www.python.org/download/releases/2.7/
+#. You need to open the Terminal app. There are many ways to do this. We suggest the following:
+	* Click the magnifier icon on the upper right corner and search "terminal".
+	* Press *'Enter'* with the "Terminal" app selected and it should open.
+#. In the Terminal, enter *Python -V* and press *'Enter'*. **The version number should reflect the Python version you just installed.**
+#. KA Lite uses Git to distribute it's code for non-Windows operating systems.
```

Comment:
OS X already has a builtin git, so I'd prefer if we removed this step to lessen the number of programs our user has to install: http://stackoverflow.com/questions/8957862/how-to-upgrade-git-to-latest-version-on-mac-os-x